workspaces/adr/plugins/adr-backend/package.json

```diff
@@ -35,6 +35,7 @@
   },
   "dependencies": {
     "@backstage-community/plugin-adr-common": "workspace:^",
+    "@backstage-community/search-backend-module-adr": "workspace:^",
```

Comment:
![Optional: FYI](https://gh-labels.dancrews.com/optional/fyi.svg)
This is currently used [here](https://github.com/backstage/community-plugins/blob/90567e287ba1eb2d0353689b76a61624282b98ca/workspaces/adr/plugins/adr-backend/src/deprecated.ts#L1-L4), but not installed.